sdk/compiler/damlc/daml-stdlib-src/DA/Time.daml

```diff
@@ -92,3 +97,18 @@ instance Bounded Time where
     minBound = microsecondsSinceEpochToTime (-62135596800000000)
     maxBound = microsecondsSinceEpochToTime 253402300799999999
 
+-- | True iff the ledger time of the transaction is less than the given time.
+isLedgerTimeLT: Time -> Update Bool
+isLedgerTimeLT = primitive @"ULedgerTimeLT"
+
+-- | True iff the ledger time of the transaction is less than or equal to the given time.
+isLedgerTimeLE: Time -> Update Bool
+isLedgerTimeLE time = isLedgerTimeLT (addRelTime time (microseconds 1))
```

Comment:
@remyhaemmerle-da @carlpulley-da : are you sure that this will never throw?

Reply:
Please add a test that uses `time = maxBound`.

Reply:
Good catch. Forgot about that case!